ci-operator/templates/openshift/installer/cluster-launch-installer-metal-e2e.yaml

```
@@ -540,11 +542,15 @@ objects:
         update_image_registry &
 
         echo "Completing UPI setup"
-        openshift-install --dir=${ARTIFACT_DIR}/installer wait-for install-complete 2>&1 | grep --line-buffered -v password &
+        openshift-install --dir=${TMP_SHARED}/installer wait-for install-complete 2>&1 | grep --line-buffered -v password &
         wait "$!"
 
         # Password for the cluster gets leaked in the installer logs and hence removing them.
-        sed -i 's/password: .*/password: REDACTED"/g' ${ARTIFACT_DIR}/installer/.openshift_install.log
+        sed '
+          s/password: .*/password: REDACTED/;
+          s/X-Auth-Token.*/X-Auth-Token REDACTED/;
+          s/UserData:.*,/UserData: REDACTED,/;
```

Should these substitutions be global?
```suggestion
          s/password: .*/password: REDACTED/g;
          s/X-Auth-Token.*/X-Auth-Token REDACTED/g;
          s/UserData:.*,/UserData: REDACTED,/g;
```

---

I don't see why it couldn't.  wking had suggested I update the original redaction with the code from the multi-stage installer step here, https://github.com/openshift/release/blob/9a726f0def12b3e735b38d74a95cd9a21c84720c/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L63-L67.  Upon searching the release repo I'm seeing instance where global is used so there is no consistency.  I'll update.

---

@staebler Updated.